kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStateListRow.js

```diff
@@ -22,31 +23,33 @@ define([
         let container, ui, jobId, jobState;
         const widgets = {};
 
-        function _createStatusLabel(jobStateObject) {
-            return (
-                span({
-                    class: `fa fa-circle ${cssBaseClass}__icon--${jobStateObject.status}`,
-                }) +
-                ' ' +
-                Jobs.jobLabel(jobStateObject, true)
+        function _updateStatusLabel(jobStateObject) {
+            const statusLabel = Jobs.jobLabel(jobStateObject, true);
+            ui.setContent(
+                'job-state',
+                div(
+                    {
+                        dataToggle: 'tooltip',
+                        dataPlacement: 'bottom',
+                        title: statusLabel,
+                    },
+                    [
+                        span({
+                            class: `fa fa-circle ${cssBaseClass}__icon--${jobStateObject.status}`,
+                        }),
+                        statusLabel,
+                    ]
+                )
             );
         }
 
-        function _createActionButton(jobStateObject) {
-            return div(
-                {
-                    class: `${cssBaseClass}__cell_action--${jobStateObject.status}`,
-                    role: 'button',
-                    // TODO: these actions need to be added
-                    id: events.addEvent({
-                        type: 'click',
-                        handler: function () {
-                            // handle action button clicks
-                        },
-                    }),
-                },
-                [Jobs.jobAction(jobStateObject)]
-            );
+        function _updateActionButton(jobStateObject) {
```

Comment:
change this to use a button instead of a div, and add in data attributes that will encode what the button press should do.